CarCatcher

harshDhiman3344 opened the pull request:
# Your checklist for this pull request

## Author name

**Author:**

Hoshiko

<!-- A name or nickname that you want to appear as the author of the game -->

## About your game

**What is your game about?**

This game is about catching a falling kitten which appears randomly on different places.

<!-- Example: Pushing boxes to the goal. (from [Sokoban Plus](https://sprig.hackclub.com/gallery/sokoban_plus)) -->

**How do you play your game?**

The player can be moved left and right using A and D to go under the kitten and catch it

<!-- Example: Press WASD to move, J to restart and K to toggle trails, Get A boxes (cyan) to A goals (green), Get B boxes (magenta) to B goals (red), Get normal boxes (gray) to either goal. (from [Sokoban plus](https://sprig.hackclub.com/gallery/sokoban_plus)) -->

## Code
Check off the items that are true. <!-- Put a `x` in the  `[ ]`. Example: `[x] The game was made...` -->
- [x] The game was made using the [Sprig editor](https://sprig.hackclub.com/editor).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [ ] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [ ] The name of the image matches the name of your file. <!-- Example: `sokoban_plus.js` -> `sokoban_plus.png`. -->

> Thanks for your PR!

GalaxyGamingBoy commented:
Hey! Please update your games metadata to match the following format. All data must be current and about your game only. Thanks :)

Paste this code at the top of your game file fill it out with your game's information where MM is the Current Month (ie 08) and DD is the Current Day (ie 03).
```
/*
@title:
@author: 
@description:
@tags: []
@addedOn: 2025-MM-DD 
*/
```

harshDhiman3344 commented:
Hey!, added the metadata as requested :D

GalaxyGamingBoy commented:
Thank you!! Everything metadata wise seems good :) A reviewer will take a closer look soon

whatwareweb commented:
this game is really neat, and i like the music!!!

2 things to add a little polish before we can merge:

whatwareweb reviewed (changes requested):
requesting changes as described above

harshDhiman3344 commented:
Sure. working on it!

harshDhiman3344 commented:
Hey!!, i added the option to restart the game by pressing "S" after its game over and changed the background to a blue sky with a few clouds.
Kindly re review the changes.
